lib/stat_tracker.rb

```diff
@@ -264,12 +324,24 @@ def lowest_scoring_home_team
 
 # ==================       Season Stats Methods      ==================
 
+  def winningest_coach(season_id)
+    find_all_wins_by_coach(season_id).max_by do |coach, percent_wins|
+      percent_wins
+    end[0]
+  end
```

Comment:
Should this be finding the max of _percent_wins_by_coach_ as opposed to _find_all_wins_by_coach_?